### PR TITLE
Add vnc example with xorg desktop environment

### PR DESCRIPTION
**Pull request by afbjorklund**
Set automatic login using a simple login manager (slim), and basic xorg programs including xterm and openbox wm.

Add a panel and some programs for the applications menu, use tint2 instead of the removed fbpanel and basic apps.

lima-vnc

Also works with https://novnc.com/

And with  [lima-gui ](https://github.com/afbjorklund/lima-gui) "Display" button

----

* https://github.com/lima-vm/lima/discussions/1002
* https://github.com/lima-vm/lima/pull/1899

### CODE REVIEW

File: examples/experimental/vnc.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- location: "https://cloud-images.ubuntu.com/releases/23.04/release/ubuntu-23.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+
+mounts:
+- location: "~"
+- location: "/tmp/lima"
+  writable: true
+
+video:
+  display: "vnc"
```
Comment (AkihiroSuda) [inline examples/experimental/vnc.yaml:24]: Needs vmType: qemu

File: examples/experimental/vnc.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+provision:
+- mode: system
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    command -v Xorg >/dev/null 2>&1 && exit 0
+    # x-terminal-emulator x-session-manager x-window-manager
+    apt-get install -y xorg xterm openbox slim hsetroot tint2
```
Comment (AkihiroSuda) [inline examples/experimental/vnc.yaml:33]: DEBIAN_FRONTEND=noninteractive

**Comment by afbjorklund** [comment]
I was thinking to add a local example (display: default), for comparison... Probably using a regular desktop, like [MATE](https://ubuntu-mate.org/).

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**